packages/core/src/classes.ts

```diff
@@ -100,4 +100,7 @@ export interface Space extends Doc {
   members: Arr<Ref<Account>>
 }
 
-export interface Account extends Doc {}
+export interface Account extends Doc {
+  name: string
```

Comment:
name is full name? Should it be structured and optional? It will be not so many changes, and will be more useful?